examples/transcription/websocket/callback_stream.rs

```diff
@@ -0,0 +1,53 @@
+use std::env;
```

Comment:
This example seems generally harder to set up than other more self-contained ones since it requires the user to have access to a public url that can receive callbacks (or a non-public URL and a self-hosted instance of deepgram).  Maybe add some documentation at the top here suggesting that the caller can provide a presigned POST URL to create an S3 object.

Reply:
I'm not sure if the presigned POST will work, since we will be sending multiple POST requests back. I just used `ngrok` - would posting a link to their docs be sufficient?

Reply:
Oh does it send a separate POST for each WS message.  That's awkward.  ngrok would only work if you have a self-hosted instance, right?  Maybe we should just leave it alone.

Reply:
Yep, separate POST request each message. And yea, have to setup a local HTTPserver locally, a bit of a pain.